contracts/koh/contracts/KingAutomaton.sol

```diff
@@ -655,96 +741,57 @@ contract KingAutomaton {
   // DEX
   //////////////////////////////////////////////////////////////////////////////////////////////////
 
-  uint256 public minOrderETH = 1 ether / 10;
-  uint256 public minOrderAUTO = 1000 ether;
+  DEX.Data public dex_data;
 
-  enum OrderType { Buy, Sell, Auction }
+  uint256 public constant minOrderETH = 1 ether / 10;
+  uint256 public constant minOrderAUTO = 1000 ether;
 
-  struct Order {
-    uint256 AUTO;
-    uint256 ETH;
-    address payable owner;
-    OrderType orderType;
-  }
-
-  Order[] public orders;
-
-  function getDEXBalance() public view returns (uint256) {
-    return balanceOf(DEXAddress);
-  }
-
-  function removeOrder(uint256 _id) private {
-    orders[_id] = orders[orders.length - 1];
-    orders.pop();
+  function withdraw(uint256 _value) external {
```

Comment:
I understand this is a good practice, but in our case it's mainly adding gas costs. I'd like to revert to sending the value directly instead of allocating a slot and then having to make another call.